coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -121,6 +125,17 @@ def _parse_documentation_with_symbols(self, param_identifiers,
                 param_desc = splitted[1]
                 parsed.append(self.Parameter(name=cur_param, desc=param_desc))
 
+            elif stripped_line.startswith(excp_identifiers[0]):
+                parse_mode = self.ExceptionValue
+                excp_offset = line.find(
+                    excp_identifiers[0]) + len(excp_identifiers[0])
```

Comment:
Oh yes! that also has a case, what if there's an indentation in metadata's(params,exceps...) desc.
like
```
    """
    This is good

    :param xyz:
        param definition
            1. no idea if someone will write like this.
            2. model point
    """
```
`line` is use to preserve the indentation instead of `stripped_line`.